doc/Database_Setup_and_Migrations.md

```diff
@@ -0,0 +1,92 @@
+# How the database setup and migration works
+
+The Fundraising Application has two different database environments:
+
+- The **local development environment**. We're running a MariaDB database in a Docker container for local user and acceptance testing. We are using an in-memory SQLite database for running unit tests. This makes running the tests fast and allows for a clear separation between interactive testing and automated testing. 
+- The **server environment** has a dedicated database server with two databases (test and production) that our test and production web servers connect to. The Fundraising App and the Fundraising Operation Center both share the same production or testing database.
```

Comment:
```suggestion
- The **server environment** is running on a dedicated database server with two databases (test and production) that our test and production web servers connect to. The Fundraising App and the Fundraising Operation Center both share the same production or testing database.
```

Reply:
Hmm, maybe "consists of" or just "uses"? Pinging @Abban for a comment

Reply:
"Consists of" is good